.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Can we keep this? :)

```suggestion
blank_issues_enabled: true
```